system/lib/libc/musl/ldso/dynlink.c

```diff
@@ -858,7 +858,7 @@ static void *dl_mmap(size_t n)
 	void *p;
 	int prot = PROT_READ|PROT_WRITE, flags = MAP_ANONYMOUS|MAP_PRIVATE;
 #ifdef SYS_mmap2
-	p = (void *)__syscall(SYS_mmap2, 0, n, prot, flags, -1, 0);
+	p = (void *)__syscall(SYS_mmap2, 0, n, prot, flags, -1, 0, 0);
```

Comment:
this change is in general musl code, not in the emscripten-specific part - why is it needed?

Reply:
Ah, you're right. It isn't needed. I've now reverted this change.